client/store/checkUsername.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+
+export const RESPONSE_MESSAGE = 'RESPONSE_MESSAGE'
```

Comment:
@AnnaC91 i'm not sure what `Response_message` refers to ..  response message is for when there is an error? so should we replace this with 
`USERNAME_ERROR_MESSAGE`  or `ERROR_MESSAGE`